.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -27,6 +27,7 @@ template_path=.github/ISSUE_TEMPLATE/bug_report.md
   * Firebase SDK version: _____
   * Firebase Component: _____ (Auth, Core, Database, Firestore, Messaging, Storage, etc)
   * Component version: _____
+  * Destination platform: `iOS/iPadOS | Native macOS | Mac Catalyst | tvOS | watchOS` (select all that apply)
```

Comment:
Should we also have a yes/no/N/A for repros in Simulator?

Reply:
Maybe we could write this as

```
Select all targets affected by this issue:
- [ ] iOS/iPadOS
- [ ] macOS (AppKit)
- [ ] macOS (Catalyst)
- [ ] tvOS
- [ ] watchOS
- [ ] App Clips
- [ ] Extensions
- [x] iOS Simulator
```